gitk-git/gitk

```diff
@@ -2,7 +2,7 @@
 # Tcl ignores the next line -*- tcl -*- \
 exec wish "$0" -- "$@"
 
-# Copyright © 2005-2016 Paul Mackerras.  All rights reserved.
+# Copyright � 2005-2016 Paul Mackerras.  All rights reserved.
```

Comment:
I was curious what was changed on this line. There seems to be some encoding issue with the copyright symbol. The symbol looks unchanged here but `git diff` in CLI shows a sequence `<A9>` instead of it.

```diff
$ git diff chiga17/master chiga17/gitk-fetch-option
...
-# Copyright © 2005-2016 Paul Mackerras.  All rights reserved.
+# Copyright <A9> 2005-2016 Paul Mackerras.  All rights reserved.
```

Reply:
Looks like some encoding issue with some old editor.
I have reverted this line.